README.md

```diff
@@ -1,24 +1,160 @@
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Ruby on rails</a></li>
+  </ul>
 
-* Services (job queues, cache servers, search engines, etc.)
+  <summary>Database</summary>
+  <ul>
+    <li><a href="#">PostgreSQL</a></li>
+  </ul>
 
-* Deployment instructions
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+    <li><a href="#">Redux</a></li>
+  </ul>
+</details>
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **[Display greetings - RoR_React on page load]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Preriquisites
+
+In order to run this project you need:
+
+- **Ruby 3.2.2** installed
+
+- **Rails** installed
+
+- **Nodejs** installed
+
+- **yarn** installed
+
+Run this command to install **Rails**
+
+```bash
+$ gem update --system && gem install rails && gem install pg
+```
+
+## Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```bash
+$ git clone https://github.com/mahabubx7/hello-rails-react.git
+$ cd hello-rails-react
+```
+
+To install dependencies, run:
+
+```bash
+$ bundle install
+$ yarn install
+```
+
+**Set up the database** <br>
+
+To create the database, run:
+
+```bash
+$ rails db:create
+```
+
+To create/update the schema/database, run:
+
+```bash
+$ rails db:migrate
+$ rails db:seed
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mahabub Alam**
+
+- GitHub: [@mahabubx7](https://github.com/mahabubx7)
+- LinkedIn: [in/mahabubx7](https://linkedin.com/in/mahabubx7)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add button to generate greeting]**
+- [ ] **[Add navigation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Leeoasis/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a star please!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+ <ul>
+    <li>
+        Microverse Inc.
+    </li>
+  </ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly fix this broken link. It seems the file you're referencing to does not exist.

![license](https://github.com/mahabubx7/hello-rails-react/assets/26597432/9f3d13ad-32d3-485d-99e4-a5555ebe6565)